playMotionSound/cutSounds.m

```diff
@@ -0,0 +1,96 @@
+% (C) Copyright 2017 Stephanie Cattoir
+% (C) Copyright 2021 CPP LePoulpe developers
+
+function [soundChunks] = cutSounds(nbSpeakers, saveAsWav)
```

Comment:
may I ask why do you want to have this output? it is not human-readable, not easy to understand if script worked or not neither.

Reply:
btw, "25 speakers" in the input file name is deeply confusing.

Reply:
(1) The output `soundChunks` would be usefull in the future if one decides to not cut and save wav files but just cut a play an audio file directly. Regarding the non-human readable what do you mean? Any suggestion?

(2) "25 speakers" what do you mean?

Reply:
I do not know why the input file is names "600_pn_25speakers", thus I lost 5min to understand why 25speakers instead of 31... I guess you or Steph meant something with it but...

Reply:
that was a sound taken from the old codebase we have and kept the name as it was